### DISCUSSION TITLE
Only allow an add-on product to be bought during the order creation

### DISCUSSION THREAD

**Discussion by MaLund13** [pretix/pretix#4417] [Ideas]
Hi,

I'm using pretix to sell a multi-day trip where we also offer add-on products such as meals. As the trip goes on sale a few months in advance, our customers might have different needs of flexibility. I'd therefore like to offering better/more flexible cancellation terms as an optional add-on, kind-of like a cancellation insurance.

The issue that I've run into while trying to add this is that there doesn't seem to be a **per-product or per-category** setting for whether an add-on can be bought **after** the point of the original purchase (order creation). Preventing this is essential as it changes the terms of the original sale.

Ideally, the option to change/add add-on products (as can be set in Settings -> Cancellation -> Order changes) should be kept enabled, so that customers can, at a later point, go in and add other add-ons through their order page should they wish to do so. However, I want the "flexible booking" to only be visible and possible to add when making the original booking (i.e., during the checkout flow for creating the order), and not when the user's making a self-service order change after order creation.

Ideally what I'm looking for is a product availability setting "Allow product to be purchased after order creation", which I'd then disable to get my desired behavior. The setting could also be on category-level.

While writing this up, I've realized that one option to achieve this currently is with product variations on the main products, instead of using add-ons at all. But I'm still curious if it'd be possible to somehow do this using add-ons? For the customer perspective, that feels like it would be a more intuitive approach.

Thanks!

**Comment by raphaelm** [comment]
No, this is not possible using addons :(
